demo/src/main/java/com/andrefrsousa/supertoolbar/demo/DemoActivity.kt

```diff
@@ -19,38 +19,34 @@ class DemoActivity : AppCompatActivity() {
 
         toolbar.title = getString(R.string.app_name)
 
-        val layoutManager = LinearLayoutManager(this)
-        items_list.layoutManager = layoutManager
-        items_list.adapter = ItemsAdapter()
-
-        items_list.addOnScrollListener(object : RecyclerView.OnScrollListener() {
-            override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
-                super.onScrolled(recyclerView, dx, dy)
-                toolbar.setElevationVisibility(recyclerView.canScrollVertically(-1))
-            }
-        })
+        items_list.apply {
```

Comment:
Since you are storing the result of the apply, you should use **run** instead.

Reply:
Exactly, return a copy of that object. If you want only to update the object its properties you should use RUN in this case. 

Apply is used for example:

```
val test = items_list.apply {
}
```